src/accordion/accordion.ts

```diff
@@ -130,6 +145,9 @@ export class NgbAccordion implements AfterContentChecked {
       this._closeOthers(this.activeIds[0]);
       this._updateActiveIds();
     }
+
+    // update classes for panels
+    this._setClassType();
```

Comment:
Why do we do this in `ngAfterContentChecked`? This doesn't sound right. We should simply calculate proper class in a template.